README.md

```diff
@@ -38,7 +38,7 @@ terminal (Mac OS), `oh-my-zsh` with `powerlevel9k` theme and `powerline nerd-fon
   ```sh
   function lc()
   {
-    ruby /path/to/colorls/colorls.rb $1;
+    ruby ~/bin/colorls/colorls.rb $1;
```

Comment:
I would advise to do that via `alias lc='lc -r'` to not clutter the function definition.